### PR TITLE
Add an 'isReserved()' API to check for reserved IPv6 addresses

### PR DESCRIPTION
**Pull request by sdmodi**
Reviewer: @vaterlaus @andi-bigswitch

### CODE REVIEW

**Comment by bsn-abat** [comment]
Find artifact changes from this pull request at https://github.com/floodlight/loxigen-artifacts/tree/pr-583-jenkins-loxigen_pull_req_artifacts-145

**Comment by bsn-bot3** [comment]
Build finished.

**Comment by bsn-bot3** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins.bigswitch.com/job/loxigen_pull_req_artifacts/145/

**Comment by bsn-bot3** [comment]
Build finished.

**Comment by bsn-bot3** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins.bigswitch.com/job/loxigen_pull_req_docker/157/

File: java_gen/pre-written/src/test/java/org/projectfloodlight/openflow/types/IPv6AddressTest.java [hunk 1]
```diff
@@ -436,6 +436,85 @@ public void testOfExceptions() throws Exception {
...
     }
 
+    @Test
+    public void testReservedIp() {
+        IPv6Address addr;
+        /* fec0::/10 Reserved by IETF [RFC3879]. Deprecated by
+         * [RFC3879] in September 2004. Formerly a Site-Local scoped
+         * address prefix.
+         */
+        addr = IPv6Address.of("fec0::10");
```
Comment (andi-bigswitch) [inline java_gen/pre-written/src/test/java/org/projectfloodlight/openflow/types/IPv6AddressTest.java]: would be good to also test, for each of the rules, that the first address that lies outside of the reserved range is correctly reported as not-reserved.

Comment (sdmodi) [reply java_gen/pre-written/src/test/java/org/projectfloodlight/openflow/types/IPv6AddressTest.java]: I have added a few more tests.

**Comment by sdmodi** [comment]
SELF-ACK

**Comment by bsn-abat** [comment]
ABAT: START_MERGE:  https://jenkins.bigswitch.com/job/loxigen-abat-automerge/882//console

**Comment by bsn-bot3** [comment]
Build finished.

**Comment by bsn-bot3** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins.bigswitch.com/job/loxigen_pull_req_docker/158/

**Comment by bsn-abat** [comment]
Find artifact changes from this pull request at https://github.com/floodlight/loxigen-artifacts/tree/pr-583-jenkins-loxigen_pull_req_artifacts-146

**Comment by bsn-bot3** [comment]
Build finished.

**Comment by bsn-bot3** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins.bigswitch.com/job/loxigen_pull_req_artifacts/146/

**Comment by bsn-abat** [comment]
ABAT: ACCEPT:  Successfully merged

In case you want to see the build log, check out :
https://jenkins.bigswitch.com/job/loxigen-abat-automerge/882//console
/cc

**Pull request closed without merging**